joi/joi.js

```diff
@@ -0,0 +1,21 @@
+const Joi = require('joi');
+
+
+const joiIssueSchema = Joi.object({
+    title: Joi.string().required(),
+    text: Joi.string().required(),
+    user: Joi.string().required(),
+    closed: Joi.boolean().required(),
+    file: Joi.array(),
```

Comment:
Do we need to validate file objects?

Reply:
Should I just use Joi.any()?

Reply:
Why any? file object has predictable structure?

Reply:
Well it has, I think. It should be an array of file names.